nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -102,6 +103,45 @@ scenarioTypes {
       classPath: [${modelDir}"/defaultModel.jar", ${liteComponentsDir}"/liteRequestResponse.jar", ${liteComponentsDir}"/liteBase.jar"]
     }
   }
+  "lite-streaming-k8s": {
+    deploymentConfig {
+      type: "lite-streaming-k8s"
+      configExecutionOverrides {
```

Comment:
please rember to add todo that it should be made more human firendly in the next change ;)

Reply:
:+1: